.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,43 @@
+name: "CodeQL Advanced"
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+  schedule:
+    - cron: '22 16 * * 5'
```

Comment:
Specific time and day of the week, issue in multiple related PRs.

Reply:
I shall make the changes on all the PR's now 👍